packages/react-context/src/__tests__/user-profile.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { useUserProfileContext, UserProfileContext } from '../user-profile';
+
+const TestComponent: React.FC<{ index?: number }> = () => {
```

Comment:
TBH I think we don't need the wrapping hook? And then this test is literally just testing React context, so it can go away? Or I guess to preserve code coverage the one thing we could test (and do test on other contexts) is the expected default value